test/e2e/app_management_test.go

```diff
@@ -373,6 +373,49 @@ func TestKsonnetApp(t *testing.T) {
 		})
 }
 
+func TestKustomize2AppSource(t *testing.T) {
+
+	patchLabelMatchesFor := func(kind string) func(app *Application) {
```

Comment:
do you want to move this to `expectations.go`?

Reply:
Is it generic enough? Otherwise, sure!